test/e2e/util/util.go

```diff
@@ -46,8 +46,8 @@ func initE2E() *TestE2eConfig {
 		e2eConf.clusteradm.h = HandledOutput{}
 
 		// wait for resources to terminate
-		t := 30
-		fmt.Fprintf(os.Stdout, "wait resources to be terminated for %d seconds", t)
+		t := 60
```

Comment:
it is better that we poll on whether ns still exists?

Reply:
done